app/src/common/lang.js

```diff
@@ -167,19 +167,16 @@ const supportedLanguages = {
   },
   "ko-KR": {
     langName: "한국어 (대한민국)",
-    welcomeMessage: "안녕하세요! 내가 어떻게 도움이 될 수 있습니다?",
-    inputPlaceholder: "무엇이든 물어보세요...",
-    listeningMessage: "청취...",
+    welcomeMessage: "안녕하세요? 무엇을 도와드릴까요?",
+    inputPlaceholder: "무엇이든 물어보세요",
+    listeningMessage: "네, 무엇을 도와드릴까요?",
     initSuggestions: [
-      { label: "날씨", query: "오늘의 날씨는 어떻습니까?" },
-      { label: "동전을 던지세요", query: "동전을 던지세요" },
-      {
-        label: "당신은 무엇을 할 수 있나요?",
-        query: "당신은 무엇을 할 수 있나요?",
-      },
+      { label: "날씨가 어때?", query: "날씨가 어때?" },
+      { label: "동전 던져 줘", query: "동전 던져 줘" },
+      { label: "무엇을 도와줄 수 있니?", query: "무엇을 도와줄 수 있니?" },
```

Comment:
The above Korean sentence **"무엇을 도와줄 수 있니?"** translates to **"What can you help?"** in English as per _Google Translate_ which contains minor grammatical mistake. Is the above translation correct as per the Korean grammar though? Just need a confirmation that the phrase preserves the meaning and has correct grammar :)

**EDIT:** The sentence should loosely translate to "How can you help?", "What can you help with?", etc. You may refer to the English translation for a hint.

Reply:
@Melvin-Abraham  `무엇을 도와줄 수 있니?` is from official app. `할 수` instead of `도와줄 수` would be good if the English meaning is important, but `도와줄 수` feels much warmer.

Reply:
If it's correct as per the Korean grammar, then it's fine!